doc/src/how-to/vscodeext-how-to-debug-apps-qml.qdoc

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2025 The Qt Company Ltd.
+// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GFDL-1.3-no-invariants-only
+
+/*!
+    \page vscodeext-how-debug-apps-qml.html
+    \previouspage vscodeext-how-tos.html
+    \nextpage vscodeext-reference.html
+
+    \ingroup vscodeext-how-to-qml
+
+    \title Debug Qt Quick applications
+
+    You can debug Qt Quick applications that contain both QML and C/C++ code with
+    the QML debugger and the debugger for your operating system. Usually, you use
+    GDB on Linux and \macos or the Visual Studio Windows debugger on Windows.
+
+    \note Debugging requires opening a socket at a TCP port, which presents a
+    security risk. Anyone on the Internet could connect to the application that
+    you are debugging and execute any JavaScript functions. Therefore, you must
+    make sure that the port is properly protected by a firewall.
+
+    \section1 Debug QML code
+
+    To only debug the QML code in an application, it is faster to attach the
+    debugger to a running application using a fixed port number than to have
+    \QVSC first start the application and then attach the QML debugger to it.
+
+    To debug only QML code:
+
+    \list 1
+        \li Create a \e {launch.json} file for the project. For more information,
+            see \l {Debug Qt applications}.
+        \li Select \uicontrol {Add Configuration}, and then select the
+            \uicontrol {Qt: QML: Attach by port} debug configuration.
+        \li In the \uicontrol Terminal, start the application with the following
+            arguments:
+            \badcode
+            <your_executable_path> -qmljsdebugger=host:<IP_address>,port:<port_number>,block,services:DebugMessages,QmlDebugger,V8Debugger,QmlInspector
+            \endcode
+            Where \e port (mandatory) is the port number fetched by the
```

Comment:
`Qt: Acquire Port` is only used with `Debug mixed C/C++ and QML code(Compound)`. I think it would be confusing for users to mention in this section.